### PR TITLE
Set up Github Actions

### PR DESCRIPTION
**Pull request by albertyw**
Depends on #169 

Part of #166

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    name: Build utility functions
+    steps:
+      - name: Install
+        run: |
+          pip install tox
+      - name: Run utility functions
+        run: |
+          tox -e cover
+          tox -e flake8
+          tox -e docs
```
Comment (abhinav) [inline .github/workflows/python.yml:47]: You still need a Python setup for this, IIRC.

**Review by abhinav — APPROVED** [review]
I don't remember how readthedocs publishing works, but that might need a look into -- unless it's automatic.

File: .github/workflows/python.yml (hunk 1 above)
Comment (albertyw) [reply .github/workflows/python.yml:47]: I don't think I can trigger Github Actions on this repository until this PR is merged, but this Github Actions was successful on my fork: https://github.com/albertyw/thriftrw-python/actions/runs/4213241941

**Comment by albertyw** [comment]
> I don't remember how readthedocs publishing works, but that might need a look into -- unless it's automatic.

I'm going to consider that out of scope for this PR.  Followup in https://github.com/thriftrw/thriftrw-python/issues/172

**Pull request merged**